runtime.go

```diff
@@ -38,4 +38,8 @@ type Runtime interface {
 	Delete(context.Context, Container) (*Exit, error)
 	// Events returns events for the runtime and all containers created by the runtime
 	Events(context.Context) <-chan *Event
+	// Pause handles the pause action for a container within the runtime
+	Pause(context.Context, Container) error
```

Comment:
Is there a reason why you placed these methods on the `Runtime` and not on the `Container` interfaces?